vividus-plugin-electron/src/main/resources/properties/profile/web/desktop/chrome/electron/profile.properties

```diff
@@ -0,0 +1,2 @@
+web.driver.CHROME.binary-path=
```

Comment:
`properties/profile/web/desktop/chrome/electron/profile.properties` -> `properties/profile/desktop/electron/profile.properties` ?

Reply:
Could you please clarify, how should I manage all the web default properties? Copy-paste them from root web profile.properties?

Reply:
multiple profiles? the PR was merged to `master`

Reply:
For me, it seems unnatural if you wanna test electron app you should add chrome and don't miss the order of the profiles, it's a bit too complicated isn't?